src/lib/pci_virtio_block.c

```diff
@@ -115,7 +115,7 @@ struct virtio_blk_hdr {
  * Debug printf
  */
 static int pci_vtblk_debug;
-#define DPRINTF(params) if (pci_vtblk_debug) printf params
+#define DPRINTF(params) if (pci_vtblk_debug) log_info params
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.